### PR TITLE
Add ios_bundle test

### PR DESCRIPTION
**Pull request by ajanuar**
- Merged with latest upstream
- Add `ios_bundle_test.sh`

### CODE REVIEW

File: apple/internal/rule_support.bzl [hunk 1]
```diff
@@ -270,6 +270,7 @@ _RULE_TYPE_DESCRIPTORS = {
             deps_cfg = apple_common.multi_arch_split,
             mandatory_families = True,
             product_type = apple_product_type.bundle,
+            requires_signing_for_device = False,
```
[the remark this answers is not in the document]
Comment (brentleyjones) [reply apple/internal/rule_support.bzl:273]: I think it would

**Comment by brentleyjones** [comment]
After you sign the CLA I can take a look at this.

**Comment by ajanuar** [comment]
@googlebot I signed it!

**Comment by ajanuar** [comment]
> After you sign the CLA I can take a look at this.

Ok @brentleyjones thanks

**Pull request closed without merging**